src/Shipping/ShippingController.php

```diff
@@ -61,32 +61,6 @@ function() {
 		add_filter( 'pre_update_option_pickup_location_pickup_locations', array( $this, 'flush_cache' ) );
 	}
 
-	/**
-	 * Gets a list of payment method ids that support the 'local-pickup' feature.
-	 *
-	 * @return string[] List of payment method ids that support the 'local-pickup' feature.
-	 */
-	public function get_local_pickup_method_ids() {
-		$all_methods_supporting_local_pickup = array_reduce(
-			WC()->shipping()->get_shipping_methods(),
-			function( $methods, $method ) {
-				if ( $method->supports( 'local-pickup' ) ) {
-					$methods[] = $method->id;
-				}
-				return $methods;
-			},
-			array()
-		);
-
-		// We use array_values because this will be used in JS, so we don't need the (numerical) keys.
```

Comment:
Minor nitpick: This won't remove the numerical keys, but it will reset them after array_unique() passes, which is a good safeguard